.cm/change_categories.cm

```diff
@@ -216,13 +188,9 @@ platforms:
   - kotlin_dsl:
     name: 'kotlin_dsl'
     subprojects:
+      - 'platforms/core-configuration/'
```

Comment:
🤔 `core-configuration` contains only `kotlin-dsl`, but this can be changed in the future, right? `core-configuration`  doesn't match `kotlin-dsl`. IMHO, it's better to leave it as a list of folders.

Reply:
The final goal of platformization is everything under a `/platforms` dir and when we reach that there should be no need to be more specific.

My goal was 1 entry in this list of mappings per platform, and just continue to refine this as projects move, but maybe you're right that it's too soon for this.  

I'll restore the list of individual dirs.